### ISSUE TITLE
Issue with Chaospy version when pip installing EasyVVUQ

### ISSUE THREAD

**Issue by olivhoenen** [UCL-CCS/EasyVVUQ#130]
Hi there, 

when you install EasyVVUQ through pip: `pip install easyvvuq` it comes with chaospy in version 3 (3.0.7), while from what @jlakhlili told me version 2.3.5 is expected. Because of the differences in the API we end up with broken PCE sampler (tested from Fusion application):
```
python test_uq_ets.py
Traceback (most recent call last):
  File "test_uq_ets.py", line 94, in <module>
    my_sampler = uq.sampling.PCESampler(vary=vary, polynomial_order=4)
  File "/gw/switm/python/3.6/lib/python3.6/site-packages/easyvvuq/sampling/pce.py", line 96, in __init__
    sparse=sparse)
TypeError: generate_quadrature() got an unexpected keyword argument 'domain'
```

**Comment by dww100** [comment]
Is the `dev` branch version compatible with chaospy 3?

At one stage we had to fix to 2.3.5 due to another issue but we submitted fixes for `chaospy` for those. I guess this is an unexpected consequence of that.

I guess we need to patch the master branch and remake the pip branch but the choice is which way to go:
1. Ensure the old version in the setup.py
2. Update the code to use new API

I'd favour the latter all else being equal as it should be more sustainable into the future.

**Comment by jlakhlili** [comment]
The `dev` branch is compatible with `Chaospy 2` (the latest release).

**Comment by dww100** [comment]
If the pip version is higher what is the meaning of the release in this case? Is there maintenance of the 2.x branch?

Maybe this is a question for @jonathf

**Comment by jonathf** [comment]
There is maintenance on 2.X branch in the sense if some user post a bugreport, I will fix it. There will be no backporting though.

As for the name change in question, it is specific for 3.0.6, where a major refactor adding a lot of new features to quadrature. Things got moved around and the arg `domain` got renamed to `dist`. Silly bug.

If you want to fix it quickly, just rename it to match. Otherwise I can make a patch on the Chaospy side of things.

**Comment by raar1** [comment]
No problem, I've renamed ```domain``` to ```dist``` and things work fine for version 3.0.7

**Comment by raar1** [comment]
I have fixed the current chaospy requirement at 3.0.7 for the most recent release, and will increase that to a more recent chaospy version at some point, but for now I think this issue has been addressed.
